torch/_functorch/eager_transforms.py

```diff
@@ -1516,18 +1516,18 @@ def functionalize(func: Callable, *, remove: str = "mutations") -> Callable:
         ...     c.add_(1)
         ...     return b
         ...
-        >>> inpt = torch.randn(2)
+        >>> input = torch.randn(2)
```

Comment:
This is done on purpose as "input" is a reserved keyword in python

Reply:
Changed to `input_` as a common practice.